android/src/main/java/com/pdftron/pdftronflutter/helpers/PluginUtils.java

```diff
@@ -1198,6 +1214,14 @@ private static Uri getUri(Context context, String path, boolean isBase64, String
                         .showEditMenuOption(false)
                         .showReflowOption(false);
             }
+
+            if (!saveCopyOptions.isEmpty()) {
```

Comment:
the bracket location is strange, i'd imagine it to be same indentation level as hideViewModeItems

Reply:
Right, thank you